src/user/Activity/Activity.js

```diff
@@ -49,41 +52,60 @@ class Activity extends Component {
 
   render() {
     const { view } = this.state
-
+    const toggleSidebar = () => {
+      this.setState((prevState) => {
+        return {
+          sidebarOpen: !prevState.sidebarOpen
+        }
+      })
+    }
     return (
-      <div className="overall_container">
-        <div className="main_navigation">
-          <Navigation orgSettings={this.state.org} user={this.props.user} />
-        </div>
-        <div className="org_settings_view">
-          <div className="main_section">
-            <div className="left_nav">
-              <p className="header_text">Community Settings</p>
-              <LeftNav
-                data={{
-                  option: this.state.option,
-                  changeOption: this.changeOption.bind(this),
-                }}
-              />
-            </div>
-            <div className="right_section">
-              {view === "profile" ? <OrgProfile /> : null}
-              {view === "permission" ? <OrgPermission /> : null}
-              {view === "settings" ? <OrgSettings /> : null}
-              {view === "authentication" ? <OrgAuth /> : null}
-              {view === "maintenance" ? <OrgMaintenance /> : null}
-              {view === "activity" ? (
-                <Users
-                  handleOption={{ changeOption: this.changeOption.bind(this) }}
-                />
-              ) : null}
-              {view === "details" ? (
-                <ActivityTimeline />
-              ) : null}
+      <>
+        <Navigation orgSettings={this.state.org} user={this.props.user} />
+            <div className="main_section">
+              <div className="left_nav">
+                <p className="header_text">Community Settings</p>
+                <Desktop>
```

Comment:
@vaibhavdaren sir, from early days I requested to change the different views to CSS media query, instead of using a third party for different screen sizes.
@lazycipher please have a look.
cc @AuraOfDivinity

Reply:
@Rupeshiya, I'll use media queries instead and push code.

Reply:
@Rupeshiya, I've made changes!

Reply:
You have used "react-responsive" media query or pure CSS media query ??

Reply:
I've used pure CSS media queries.

Reply:
Please check "breakpoints.js" once.

Reply:
I'll remove that file and remove the package from package.json.
Upon looking a bit more, I found out that there're few places where these are being used.
How about solving this in another PR and merge this one to move forward?